code-challenges.test.js

```diff
@@ -13,36 +14,84 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
+
 // a) Create a test with expect statements for each of the variables provided.
 
+//Pseudocode:We invoke the method describe/it with the argument fibo.
+//in plain words we describe what the function does 'takes in any number greater than 2 and returns an array that length containing the numbers of the Fibonacci sequence', then we provided it with an argument for the function to call on. Using toEqual we chain it together and expect it to equal the expected output. 
+//Now that the setup is complete for the test we run it using "$ yarn jest" and it should give us an error that says fibo not defined, which is a good error. 
+
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+describe('fibo', () => {
+    it('takes in any number greater than 2 and returns an array that length containing the numbers of the Fibonacci sequence', () => {
+        expect(fibo(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+        expect(fibo(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
 
 // b) Create the function that makes the test pass.
+//Pseudocode: create a function fibo and assign a new array to 0 and 1 for the Fibonacci sequence to start.
+//We're gonna run a for loop to iterate the arguments length and set up the Fibonacci sequence. Each number it iterates is gonna get pushed to the new array and were gonna slice the array at 1 since it starts with a 0. 
+
 
+const fibo = (number) => {
+    let newArr = [0, 1]
+    for (let i = 2; i <= number; i++) {
+        newArr.push(newArr[i - 2] + newArr[i - 1])
+   }
+    return newArr.slice(1)
```

Comment:
Good work! 

tho... this slice seems inconvenient why not just start with [1,1]?